### PR TITLE
class4-2: update ruby coding style and delete files are not necessary

### PR DESCRIPTION
**Pull request by hominhhieu-2345**

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def create
     if @article.save
       redirect_to @article
     else
-      render 'new'
+      render "new"
```
Comment (khanh227) [inline app/controllers/articles_controller.rb]: đối với chuỗi thông thường thì dùng `'`, chuỗi có giá trị của biến hoặc phép tính `"`
ex: "hello #{world}"

File: app/controllers/articles_controller.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@ def update
     if @article.update(article_params)
       redirect_to @article
     else
-      render 'edit'
+      render "edit"
```
Comment (khanh227) [inline app/controllers/articles_controller.rb]: same here

File: app/views/articles/edit.html.erb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <h1>Edit article</h1>
  
-<%= render 'form' %>
+<%= render "form" %>
```
Comment (khanh227) [inline app/views/articles/edit.html.erb]: same here

File: app/views/articles/edit.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
  
-<%= link_to 'Back', articles_path %>
+<%= link_to "Back", articles_path %>
```
Comment (khanh227) [inline app/views/articles/edit.html.erb]: same here

File: app/views/articles/index.html.erb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <h1>Listing articles</h1>
 
-<%= link_to 'New article', new_article_path %>
+<%= link_to "New article", new_article_path %>
```
Comment (khanh227) [inline app/views/articles/index.html.erb]: same here

File: app/views/articles/index.html.erb [hunk 6]
```diff
@@ -13,9 +13,9 @@
   <tr>
     <td><%= article.title %></td>
     <td><%= article.text %></td>
-    <td><%= link_to 'Show', article_path(article) %></td>
-    <td><%= link_to 'Edit', edit_article_path(article) %></td>
-    <td><%= link_to 'Destroy', article_path(article), method: :delete, data: { confirm: 'Are you sure?' } %></td>
+    <td><%= link_to "Show", article_path(article) %></td>
```
Comment (khanh227) [inline app/views/articles/index.html.erb]: same here

File: app/views/articles/index.html.erb [hunk 7]
```diff
@@ -13,9 +13,9 @@
...
+    <td><%= link_to "Edit", edit_article_path(article) %></td>
```
Comment (khanh227) [inline app/views/articles/index.html.erb]: same here

File: app/views/articles/index.html.erb [hunk 8]
```diff
@@ -13,9 +13,9 @@
...
+    <td><%= link_to "Destroy", article_path(article), method: :delete, data: { confirm: "Are you sure?" } %></td>
```
Comment (khanh227) [inline app/views/articles/index.html.erb]: same here

File: app/views/articles/new.html.erb [hunk 9]
```diff
@@ -1,5 +1,5 @@
 <h1>New article</h1>
 
-<%= render 'form' %>
+<%= render "form" %>
```
Comment (khanh227) [inline app/views/articles/new.html.erb]: same here

File: app/views/articles/new.html.erb [hunk 10]
```diff
@@ -1,5 +1,5 @@
...
 
-<%= link_to 'Back', articles_path %>
+<%= link_to "Back", articles_path %>
```
Comment (khanh227) [inline app/views/articles/new.html.erb]: same here

File: app/views/articles/show.html.erb [hunk 11]
```diff
@@ -12,7 +12,7 @@
 <%= render @article.comments %>
 
 <h2>Add a comment:</h2>
-<%= render 'comments/form' %>
+<%= render "comments/form" %>
```
Comment (khanh227) [inline app/views/articles/show.html.erb]: same here

File: app/views/articles/show.html.erb [hunk 12]
```diff
@@ -12,7 +12,7 @@
...
 
-<%= link_to 'Edit', edit_article_path(@article) %>
-<%= link_to 'Back', articles_path %>
+<%= link_to "Edit", edit_article_path(@article) %>
+<%= link_to "Back", articles_path %>
```
Comment (khanh227) [inline app/views/articles/show.html.erb]: same here

File: app/views/comments/_comment.html.erb [hunk 13]
```diff
@@ -9,5 +9,5 @@
 </p>
 
 <p>
-  <%= link_to 'Destroy Comment', [comment.article, comment], method: :delete, data: { confirm: 'Are you sure?' } %>
+  <%= link_to "Destroy Comment", [comment.article, comment], method: :delete, data: { confirm: "Are you sure?" } %>
```
Comment (khanh227) [inline app/views/comments/_comment.html.erb]: same here

**Review by khanh227 — CHANGES_REQUESTED** [review]

File: app/controllers/articles_controller.rb (hunk 2 above)
Comment (hominhhieu-2345) [reply app/controllers/articles_controller.rb]: I fixed It

File: app/controllers/articles_controller.rb (hunk 1 above)
Comment (hominhhieu-2345) [reply app/controllers/articles_controller.rb]: @khanh227 I'm clearly and I have fixed it

File: app/views/articles/edit.html.erb (hunk 3 above)
Comment (hominhhieu-2345) [reply app/views/articles/edit.html.erb]: @khanh227 I have fixed it

File: app/views/articles/edit.html.erb (hunk 4 above)
Comment (hominhhieu-2345) [reply app/views/articles/edit.html.erb]: @khanh227 I have fixed it

File: app/views/articles/index.html.erb (hunk 5 above)
Comment (hominhhieu-2345) [reply app/views/articles/index.html.erb]: @khanh227 I have fixed it

File: app/views/articles/index.html.erb (hunk 6 above)
Comment (hominhhieu-2345) [reply app/views/articles/index.html.erb]: @khanh227  I have fixed it

File: app/views/comments/_comment.html.erb (hunk 13 above)
Comment (hominhhieu-2345) [reply app/views/comments/_comment.html.erb]: @khanh227  I have fixed it

File: app/views/articles/show.html.erb (hunk 12 above)
Comment (hominhhieu-2345) [reply app/views/articles/show.html.erb]: @khanh227  I have fixed it

File: app/views/articles/show.html.erb (hunk 11 above)
Comment (hominhhieu-2345) [reply app/views/articles/show.html.erb]: @khanh227  I have fixed it

File: app/views/articles/new.html.erb (hunk 10 above)
Comment (hominhhieu-2345) [reply app/views/articles/new.html.erb]: @khanh227  I have fixed it

File: app/views/articles/new.html.erb (hunk 9 above)
Comment (hominhhieu-2345) [reply app/views/articles/new.html.erb]: @khanh227  I have fixed it

File: app/views/articles/index.html.erb (hunk 8 above)
Comment (hominhhieu-2345) [reply app/views/articles/index.html.erb]: @khanh227  I have fixed it

File: app/views/articles/index.html.erb (hunk 7 above)
Comment (hominhhieu-2345) [reply app/views/articles/index.html.erb]: @khanh227  I have fixed it

**Review by khanh227 — APPROVED** [review]

**Pull request merged**